Assets/Scripts/Scenes/InGame/Player/Item/ItemEffectScript/MotherMilkEffect.cs

```diff
@@ -0,0 +1,198 @@
+            SoundManager.Instance.PlaySe("se_tentacle00", ownerPlayerStatus.transform.position);
+            //�ꎞ�I�ɑ��A�C�e�����g�p�s�ɂ���
+            ownerPlayerStatus.UseItem(true);
+            ownerPlayerStatus.ChangeSpeed();
+            ownerPlayerItem.ChangeCanChangeBringItem(false);
+
+            yield return new WaitForSeconds(0.75f);
+
+            int _randomNum = UnityEngine.Random.Range(1, 7);
+            Debug.Log($"MotherMilk5  {_randomNum}");
+            switch (_randomNum)
+            {
+                case 1: //HP,SAN�l��50�ɕύX
+                    ownerPlayerStatus.ChangeHealth(50 - ownerPlayerStatus.nowPlayerHealth, ChangeValueMode.Heal); break;
+                case 2:
+                    Num2Effect().Forget(); break;
+                case 3:
+                    Num3Effect().Forget(); break;
+                case 4:
+                    Num4Effect(); break;
+                case 5:
+                    Num5Effect(); break;
+                case 6:
+                    Num6Effect(); break;
+                default:
+                    Debug.Log("�z��O�̒l�ł��B");
+                    break;
+            }
+            //���A�C�e�����g����悤�ɂ���
+            ownerPlayerStatus.UseItem(false);
+            ownerPlayerStatus.ChangeSpeed();
+            ownerPlayerItem.ChangeCanChangeBringItem(true);
+
+            _canUseCount--;
+            if (_canUseCount <= 0)
+            {
+                _isCanUseMilk = false;
+                ownerPlayerItem.ConsumeItem(ownerPlayerItem.nowIndex);
+                PlayerPrefs.DeleteKey(MilkCounts);
+            }
+
+        }
+
+        //1���Ԃ���SAN�l�_���[�W�𖳌��ɂ�,10�b���Ƃ�HP��1�񕜂���
+        private async UniTaskVoid Num2Effect()
+        {
+            int _count = 0;
+            var token = ownerPlayerStatus.GetCancellationTokenOnDestroy();
+            ownerPlayerStatus.ChagngeIsNotDamagedSANValue(true);
+
+            while (true)
+            {
+                _count++;
+                if (_count % 10 == 0)
+                {
+                    ownerPlayerStatus.ChangeHealth(1, ChangeValueMode.Damage);
+                }
+
+                if (_count >= 60)
+                {
+                    break;
+                }
+                await UniTask.WaitForSeconds(1f, cancellationToken: token);
+            }
+            ownerPlayerStatus.ChagngeIsNotDamagedSANValue(false);
+        }
+
+        //1���Ԃ����X�^�~�i��0�ɂ��A10�b���Ƃ�HP��1���炷
+        private async UniTaskVoid Num3Effect()
+        {
+            int _count = 0;
+            var token = ownerPlayerStatus.GetCancellationTokenOnDestroy();
+            ownerPlayerStatus.ChangeStamina(200, ChangeValueMode.Damage);
+            ownerPlayerStatus.ChangeIsCantHealStamina(true);
+
+            while (true)
+            {
+                _count++;
+                if(_count % 10 == 0)
+                {
+                    ownerPlayerStatus.ChangeHealth(1, ChangeValueMode.Damage);
+                }
```

Comment:
```suggestion
                    ownerPlayerStatus.ChangeHealth(1, ChangeValueMode.Damage);
```
アイテムの使用変更が行いました。毎秒1減少するように変更してほしいです。